src/main/kotlin/no/digdir/informasjonsforvaltning/fdk_dataset_harvester/rdf/RDFUtils.kt

```diff
@@ -0,0 +1,50 @@
+package no.digdir.informasjonsforvaltning.fdk_dataset_harvester.rdf
+
+import org.apache.jena.rdf.model.Model
+import org.apache.jena.rdf.model.ModelFactory
+import org.apache.jena.rdf.model.Property
+import org.apache.jena.rdf.model.Resource
+import org.apache.jena.rdf.model.Statement
+import org.apache.jena.sparql.vocabulary.FOAF
+import org.apache.jena.vocabulary.DCAT
+import org.apache.jena.vocabulary.DCTerms
+import org.apache.jena.vocabulary.RDF
+import org.apache.jena.vocabulary.VCARD4
+import org.apache.jena.vocabulary.XSD
+import java.io.ByteArrayOutputStream
+import java.io.StringReader
+import java.util.*
+
+enum class JenaType(val value: String){
+    TURTLE("TURTLE"),
+    RDF_XML("RDF/XML"),
+    RDF_JSON("RDF/JSON"),
+    JSON_LD("JSON-LD"),
+    NTRIPLES("N-TRIPLES"),
+    N3("N3"),
+    NOT_JENA("NOT-JENA")
+}
+
+fun jenaTypeFromAcceptHeader(accept: String?): JenaType? =
+    when (accept) {
+        "text/turtle" -> JenaType.TURTLE
+        "application/rdf+xml" -> JenaType.RDF_XML
+        "application/rdf+json" -> JenaType.RDF_JSON
+        "application/ld+json" -> JenaType.JSON_LD
+        "application/n-triples" -> JenaType.NTRIPLES
+        "text/n3" -> JenaType.N3
+        "*/*" -> null
+        null -> null
+        else -> JenaType.NOT_JENA
```

Comment:
Kunne de siste 3 unifiseres til én?

Reply:
null og */* blir default TURTLE. Et bedre navn for NOT_JENA hadde vært NOT_ACCEPTABLE, vil bare ende opp der om det blir spesifisert en mediatype som ikke støttes.

Endrer til NOT_ACCEPTABLE